internal/keepers/util.go

```diff
@@ -60,7 +60,11 @@ func (_ *cryptoShuffler[T]) Shuffle(a []T) []T {
 type sortUpkeepKeys []ktypes.UpkeepKey
 
 func (s sortUpkeepKeys) Less(i, j int) bool {
-	return string(s[i]) < string(s[j])
+	// TODO: hacky string split assuming separator
```

Comment:
can you create a ticket here to fix this.

Also this is same as https://github.com/smartcontractkit/ocr2keepers/blob/main/internal/keepers/filter.go#L80-L81 so would be better to make a common function which does this. (getBlockNumber(upkeepKey), getUpkeepID(upkeepKey))

Reply:
https://smartcontract-it.atlassian.net/browse/AUTO-1480

Reply:
nit. ideally link them in the code so that anyone who is browsing the repo can get context

Reply:
dun

Reply:
todo: remove